src/site/stages/build/process-cms-exports/schemas/raw/node-outreach_asset.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable camelcase */
+const { getFilter } = require('../../filters');
+const nodeStandardKeys = require('../standard-keys/node');
+
+module.exports = {
+  type: 'object',
+  properties: {
+    // Standard `node` key-value pairs.
+    ...nodeStandardKeys,
+    // `outreach_asset`-specific key-value pairs.
+    field_administration: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_benefits: {
+      $ref: 'GenericNestedString',
+    },
+    field_description: {
+      $ref: 'GenericNestedString',
+    },
+    field_format: {
+      $ref: 'GenericNestedString',
+    },
+    field_intro_text: {
+      $ref: 'GenericNestedString',
+    },
+    field_media: {
+      $ref: 'EntityReferenceArray',
+    },
+    field_meta_tags: {
+      $ref: 'GenericNestedString',
+    },
+    field_meta_title: {
+      $ref: 'GenericNestedString',
+    },
+    field_office: {
+      $ref: 'EntityReferenceArray',
```

Comment:
This `EntityReferenceArray` means we have to expand it, and I think it is referring to the file `node.c71e29a0-431f-44a5-ac5c-cee0c0ef0ceb.json` in the export.